src/@orb.yml

```diff
@@ -2,7 +2,9 @@ version: 2.1
 
 description: >
   Build images and push them to the Amazon Elastic Container Registry.
-  See this orb's source: https://github.com/circleci-public/aws-ecr-orb
+
+display:
+  source_url: https://github.com/CircleCI-Public/aws-ecr-orb
```

Comment:
How would we feel about adding: 
`home_url: https://aws.amazon.com/ecr/`

Reply:
@KyleTryon I think it makes sense to add it if this orb was made in partnership with AWS (I'm not sure about its origins), but if not, I'm not too sure what our policy should be. What do you think?

Reply:
AWS is a bit of a special case. We do have somewhat of a formal partnership with them even though they do not actually author their own orbs. I prefer it for consistency too.

Reply:
@KyleTryon Thanks for clarifying! I agree that we should add it in our partner orbs. I went ahead and merged this one, but we can add `home_url` in next time!